kernel/src/lib.rs

```diff
@@ -9,8 +9,13 @@
 #![feature(asm, core_intrinsics, unique, nonzero, ptr_internals)]
 #![feature(const_fn, const_cell_new, const_unsafe_cell_new, lang_items)]
 #![feature(nonnull_cast)]
+#![feature(use_extern_macros)]
 #![no_std]
 
+extern crate tock_regs;
+
+pub use tock_regs::{register_bitfields, register_bitmasks};
```

Comment:
Is it really worth introducing `another unstable` feature just for using `extern macros` over `#[macro_use(register_bitfields, register_bitmasks)]`?

OT: I don´t like this syntax for using macros confuses me with `ordinary imports` but but .........